src/transformers/models/donut/image_processing_donut.py

```diff
@@ -76,9 +76,9 @@ class DonutImageProcessor(BaseImageProcessor):
             the `preprocess` method.
         rescale_factor (`int` or `float`, *optional*, defaults to `1/255`):
             Scale factor to use if rescaling the image. Can be overridden by `rescale_factor` in the `preprocess`
-            method.
-        do_normalize:
-            Whether to normalize the image. Can be overridden by `do_normalize` in the `preprocess` method.
+            method. Whether to normalize the image. Can be overridden by `do_normalize` in the `preprocess` method.
+        do_normalize (`bool`, *optional*, defaults to `True`):
+            Whether to normalize the image.
```

Comment:
the script has some issue and we have to fix it on our side. But `Whether to normalize the image. Can be overridden by `do_normalize` in the `preprocess` method.` should be after `do_normalize`. Could you fix this manually 🙏 Thanks